corehq/apps/translations/tests/test_bulk_app_translation.py

```diff
@@ -761,6 +761,105 @@ def test_partial_upload_id_mapping(self):
         )
 
 
+class BulkAppTranslationPartialsTest(BulkAppTranslationTestBaseWithApp):
+
+    multi_sheet_headers = (
+        (MODULES_AND_FORMS_SHEET_NAME,
+         ('Type', 'menu_or_form', 'default_en', 'image_en', 'audio_en', 'unique_id')),
+        ('menu1',
+         ('case_property', 'list_or_detail', 'default_en')),
+        ('menu1_form1',
+         ('label', 'default_en', 'image_en', 'audio_en', 'video_en'))
+    )
+    multi_sheet_upload = (
+        (MODULES_AND_FORMS_SHEET_NAME,
+         (('Menu', 'menu1', 'update_case module', '', '', 'update_case_module'),
+          ('Form', 'menu1_form1', 'update_case form 0', '', '', 'update_case_form_0'))),
+        ('menu1',
+         (('name', 'list', 'Name'),
+          ('yes', 'list', 'Yes'),
+          ('yes', 'list', 'Yep'),
+          ('no', 'list', 'No'),
+          ('no', 'list', 'Nope'),
+          ('name', 'detail', 'Name'),
+          ('name', 'detail', 'Name'),
+          ('name', 'detail', 'Name'))),
+        ('menu1_form1', [])
+    )
+
+    def setUp(self):
+        """
+        Instantiate an app with AppFactory
+        """
+        # Don't use super() because we don't want BulkAppTranslationTestBaseWithApp.app
```

Comment:
The main thing `BulkAppTranslationTestBaseWithApp` adds to `BulkAppTranslationTestBase` is the app, so if this class doesn't need it perhaps it should inherit from `BulkAppTranslationTestBase`.